data/mods/gen9ssb/moves.ts

```diff
@@ -4195,12 +4195,12 @@ export const Moves: import('../../../sim/dex-moves').ModdedMoveDataTable = {
 		onHit(target, pokemon, move) {
 			this.heal(pokemon.maxhp, pokemon, pokemon, move);
 			pokemon.cureStatus();
-			this.boost({ def: 1, spd: 1 });
-			(pokemon as any).level += 5;
+			this.boost({ def: 1, spd: 1, spe: 1 });
+			(pokemon as any).level += 10;
```

Comment:
pokemon.level is read only, apparently, so its best to remove that in pokemon.ts

Reply:
that's why we cast `any` onto `pokemon` here, unless im misunderstanding what you're saying

Reply:
ah ok. that was my only concern, and if ``any`` solves it then that works